pgx-tests/src/tests/schema_tests.rs

```diff
@@ -4,14 +4,68 @@ use pgx::*;
 
 #[pgx::pg_schema]
 mod test_schema {
+    use pgx::datum::sql_entity_graph::{PgxSql, SqlGraphEntity};
     use pgx::*;
     use serde::{Deserialize, Serialize};
 
     #[pg_extern]
     fn func_in_diff_schema() {}
 
+    #[pg_extern]
+    #[pgx(sql = false)]
+    fn func_elided_from_schema() {}
+
+    #[pg_extern]
```

Comment:
This attribute macro already has several attributes, I think `#[pg_extern(sql = ...)]` is a reasonable addition to the mix and lets us avoid having the dreaded double attributes.